### ISSUE TITLE
confusing command acceptance

### ISSUE THREAD

**Issue by samdmarshall** [magicalraccoon/tootstream#200]
one of the things that continues to get me is which words are valid for a particular command. in particular, with replying to toots i end up typing “reply #” which is invalid for the purpose of composing a reply, as it only accepts “rep”. shouldn’t the full word for the action also be accepted and anything shorter just an abbreviation/alias to the command?

**Comment by craigmaloney** [comment]
Yes, we should at least have some synonyms for shortened commands like this.
